cs2log.go

```diff
@@ -360,49 +356,49 @@ const (
 	// TeamNoticePattern regular expression
 	TeamNoticePattern = `Team "(CT|TERRORIST)" triggered "(\w+)" \(CT "(\d+)"\) \(T "(\d+)"\)`
 	// PlayerConnectedPattern regular expression
-	PlayerConnectedPattern = `"(.+)<(\d+)><([\w:]+)><>" connected, address "(.*)"`
+	PlayerConnectedPattern = `"(.+)<(\d+)><([\[\]\w:]+)><>" connected, address "(.*)"`
```

Comment:
is this correct? Shouldn't it be `\[\w+\]`. Can you provide a example line?

Reply:
It doesn't matter, as it's in a group, so this just lists the allowed characters, so either way works. Here's a example line you can test it against:

`L 05/18/2023 - 14:10:10: "user<1><[U:1:2828281]><CT>" [-412 1946 -127] killed "Johnny<11><BOT><TERRORIST>" [-405 -132 -0] with "awp"`

Reply:
i'll try tomorrow, i guess it's time to merge then...

Reply:
Yeah, try that, and feel free to merge my PR.